README.md

```diff
@@ -5,7 +5,7 @@ Kubernetes.
 It focuses on being minimal, easy to use, and secure by default.
 
 Lokomotive is fully self-hosted and includes [Lokomotive
-Components](docs/concepts/components.md) adding essential functionality for production
+Components](https://kinvolk.io/docs/lokomotive/0.5/concepts/components) adding essential functionality for production
```

Comment:
Do we have to hardcode the version here? What do we do at 0.6 release? Where should commits go between the releases? I think we should have developer documentation describing new process.

Reply:
We will have a redirect from `https://kinvolk.io/docs/lokomotive/latest/concepts/components` -> `https://kinvolk.io/docs/lokomotive/0.5/concepts/components`. So all these links should have latest instead of 0.5.

Reply:
I've changed them to latest